README.md

```diff
@@ -70,6 +71,10 @@ server.resource(
     }]
   })
 );
+
+// Start receiving messages on stdin and sending messages on stdout
+const transport = new StdioServerTransport();
+await server.connect(transport);
```

Comment:
h/t @jasonkneen for [highlighting](https://github.com/modelcontextprotocol/typescript-sdk/pull/135) that this is missing